src/ngraph/runtime/cpu/mkldnn_emitter.cpp

```diff
@@ -578,6 +578,7 @@ size_t MKLDNNEmitter::build_batchnorm_forward(const mkldnn::memory::desc& input_
                                               const mkldnn::memory::desc& mean_desc,
                                               const mkldnn::memory::desc& variance_desc,
                                               const double eps,
+                                              size_t bn_total_inputs,
```

Comment:
instead of bn_total_inputs, make this into something more meaningful like 'use_computed_stats'